app/src/main/java/com/android/shelfLife/model/user/UserRepository.kt

```diff
@@ -62,16 +53,16 @@ interface UserRepository {
   suspend fun updateSelectedHousehold(selectedHouseholdUID: String)
 
   /** @param userEmails - The set of user emails to get the user IDs for. */
-  fun getUserIds(userEmails: Set<String?>, callback: (Map<String, String>) -> Unit)
+  suspend fun getUserIds(userEmails: Set<String?>): Map<String, String>
 
   /** @param userIds - The list of user IDs to get the emails for. */
-  fun getUserEmails(userIds: List<String>, callback: (Map<String, String>) -> Unit)
+  suspend fun getUserEmails(userIds: List<String>): Map<String, String>
 
   /**
    * Selects a household and saves it to the user's data. VIEW MODELS NEED TO MANUALLY SELECT THE
    * LIST OF FOOD ITEMS!!!
    *
    * @param household - The household to select.
    */
-  suspend fun selectHousehold(household: HouseHold?)
+  suspend fun selectHousehold(householdUid: String?)
```

Comment:
you have changed the parameter from the household (type Household?) to the housholdUid (String?)

Reply:
you shoukd have updated the comments of the function